# All lines beginning with `# DP:' are a description of the patch.
# DP: Description: New csb_PL locale file
# DP: Related bugs: BZ548
# DP: Dpatch author: Denis Barbier
# DP: Patch author: Andrzej Krzysztofowicz
# DP: Upstream status: BZ548
# DP: Detailed status: Modified collation rules to tailor iso14651_t1
# DP: Date: 2006-01-18

Index: glibc-2.3.5/localedata/locales/csb_PL
===================================================================
--- /dev/null
+++ glibc-2.3.5/localedata/locales/csb_PL
@@ -0,0 +1,206 @@
+comment_char %
+escape_char  /
+%
+% Kashubian Language Locale for Poland
+% by Andrzej Krzysztofowicz <ankry at mif pg gda pl>
+% Source: information from Leszek Lubecki <leszek at mml pl>
+%
+% Distribution and use is free, also for commercial purposes.
+
+LC_IDENTIFICATION
+title      "Kashubian locale for Poland"
+source     "Information from Leszek Lubecki"
+address    ""
+contact    "Andrzej Krzysztofowicz"
+email      "ankry@mif.pg.gda.pl"
+tel        ""
+fax        ""
+language   "Kashubian"
+territory  "Poland"
+revision   "0.2"
+date       "2004-09-10"
+%
+category  i18n:1997;LC_IDENTIFICATION
+category  i18n:1997;LC_CTYPE
+category  i18n:1997;LC_COLLATE
+category  i18n:1997;LC_MONETARY
+category  i18n:1997;LC_NUMERIC
+category  i18n:1997;LC_TIME
+category  i18n:1997;LC_MESSAGES
+category  i18n:1997;LC_PAPER
+category  i18n:1997;LC_NAME
+category  i18n:1997;LC_ADDRESS
+category  i18n:1997;LC_TELEPHONE
+category  i18n:1997;LC_MEASUREMENT
+END LC_IDENTIFICATION
+
+LC_CTYPE
+copy "i18n"
+END LC_CTYPE
+
+LC_COLLATE
+copy "iso14651_t1"
+
+collating-symbol <aogonek>
+collating-symbol <atilde>
+collating-symbol <eacute>
+collating-symbol <ediaeresis>
+collating-symbol <lstroke>
+collating-symbol <nacute>
+collating-symbol <ograve>
+collating-symbol <oacute>
+collating-symbol <ocirc>
+collating-symbol <ugrave>
+collating-symbol <zdot>
+
+reorder-after <a>
+<aogonek>
+<atilde>
+
+reorder-after <e>
+<eacute>
+<ediaeresis>
+
+reorder-after <l>
+<lstroke>
+
+reorder-after <n>
+<nacute>
+
+reorder-after <o>
+<ograve>
+<oacute>
+<ocirc>
+
+reorder-after <u>
+<ugrave>
+
+reorder-after <z>
+<zdot>
+
+reorder-after <U0061>
+<U0105> <aogonek>;<BAS>;<MIN>;IGNORE
+<U0104> <aogonek>;<BAS>;<CAP>;IGNORE
+<U00E3> <atilde>;<BAS>;<MIN>;IGNORE
+<U00C3> <atilde>;<BAS>;<CAP>;IGNORE
+
+reorder-after <U0065>
+<U00E9> <eacute>;<BAS>;<MIN>;IGNORE
+<U00C9> <eacute>;<BAS>;<CAP>;IGNORE
+<U00EB> <ediaeresis>;<BAS>;<MIN>;IGNORE
+<U00CB> <ediaeresis>;<BAS>;<CAP>;IGNORE
+
+reorder-after <U006C>
+<U0142> <lstroke>;<BAS>;<MIN>;IGNORE
+<U0141> <lstroke>;<BAS>;<CAP>;IGNORE
+
+reorder-after <U006E>
+<U0144> <nacute>;<BAS>;<MIN>;IGNORE
+<U0143> <nacute>;<BAS>;<CAP>;IGNORE
+
+reorder-after <U006F>
+<U00F2> <ograve>;<BAS>;<MIN>;IGNORE
+<U00D2> <ograve>;<BAS>;<CAP>;IGNORE
+<U00F3> <oacute>;<BAS>;<MIN>;IGNORE
+<U00D3> <oacute>;<BAS>;<CAP>;IGNORE
+<U00F4> <ocirc>;<BAS>;<MIN>;IGNORE
+<U00D4> <ocirc>;<BAS>;<CAP>;IGNORE
+
+reorder-after <U0075>
+<U00F9> <ugrave>;<BAS>;<MIN>;IGNORE
+<U00D9> <ugrave>;<BAS>;<CAP>;IGNORE
+
+reorder-after <U007A>
+<U017C> <zdot>;<BAS>;<MIN>;IGNORE
+<U017B> <zdot>;<BAS>;<CAP>;IGNORE
+
+reorder-end
+
+END LC_COLLATE
+
+LC_MONETARY
+copy "pl_PL"
+END LC_MONETARY
+
+LC_NUMERIC
+copy "pl_PL"
+END LC_NUMERIC
+
+LC_TIME
+abday   "<U006E><U0069><U0065>";"<U0070><U00F2><U006E>";/
+        "<U0077><U0074><U00F3>";"<U0073><U0074><U0072>";/
+        "<U0063><U007A><U0077>";"<U0070><U0069><U0105>";/
+        "<U0073><U006F><U0062>"
+day     "<U006E><U0069><U0065><U0064><U007A><U0065><U006C><U0061>";/
+        "<U0070><U00F2><U006E><U0069><U0065><U0064><U007A><U00F4><U0142><U006B>";/
+        "<U0077><U0074><U00F3><U0072><U006B>";/
+        "<U0073><U0074><U0072><U007A><U006F><U0064><U0061>";/
+        "<U0063><U007A><U0077><U0069><U00F4><U0072><U0074><U006B>";/
+        "<U0070><U0069><U0105><U0074><U006B>";/
+        "<U0073><U006F><U0062><U00F2><U0074><U0061>"
+abmon   "<U0073><U0074><U00EB>";"<U0067><U0072><U006F>";/
+        "<U0073><U0074><U0072>";"<U0142><U017C><U00EB>";/
+        "<U006D><U00F4><U006A>";"<U0063><U007A><U0065>";/
+        "<U006C><U00EB><U0070>";"<U007A><U00E9><U006C>";/
+        "<U0073><U00E9><U0077>";"<U0072><U0075><U006A>";/
+        "<U006C><U00EB><U0073>";"<U0067><U00F2><U0064>"
+mon     "<U0073><U0074><U00EB><U0063><U007A><U006E><U0069><U006B>";/
+        "<U0067><U0072><U006F><U006D><U0069><U0063><U007A><U006E><U0069><U006B>";/
+        "<U0073><U0074><U0072><U0075><U006D><U0069><U0061><U006E><U006E><U0069><U006B>";/
+        "<U0142><U017C><U00EB><U006B><U0077><U0069><U0061><U0074>";/
+        "<U006D><U00F4><U006A>";/
+        "<U0063><U007A><U0065><U0072><U0077><U0069><U0144><U0063>";/
+        "<U006C><U00EB><U0070><U0069><U0144><U0063>";/
+        "<U007A><U00E9><U006C><U006E><U0069><U006B>";/
+        "<U0073><U00E9><U0077><U006E><U0069><U006B>";/
+        "<U0072><U0075><U006A><U0061><U006E>";/
+        "<U006C><U00EB><U0073><U0074><U006F><U0070><U0061><U0064><U006E><U0069><U006B>";/
+        "<U0067><U00F2><U0064><U006E><U0069><U006B>"
+d_t_fmt "<U0025><U0061><U0020><U0025><U0064><U0020><U0025><U0062><U0020><U0025><U0059><U0020><U0025><U0054><U0020><U0025><U005A>"
+d_fmt   "<U0025><U0059><U002D><U0025><U006D><U002D><U0025><U0064>"
+t_fmt   "<U0025><U0054>"
+am_pm   "";""
+t_fmt_ampm ""
+date_fmt       "<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
+<U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
+<U0025><U005A><U0020><U0025><U0059>"
+END LC_TIME
+
+LC_MESSAGES
+copy "pl_PL"
+END LC_MESSAGES
+
+LC_PAPER
+copy "pl_PL"
+END LC_PAPER
+
+LC_NAME
+copy "pl_PL"
+END LC_NAME
+
+LC_ADDRESS
+postal_fmt    "<U0025><U0066><U0025><U004E><U0025><U0061><U0025><U004E>/
+<U0025><U0064><U0025><U004E><U0025><U0062><U0025><U004E><U0025><U0073>/
+<U0020><U0025><U0068><U0020><U0025><U0065><U0020><U0025><U0072><U0025>/
+<U004E><U0025><U0025><U007A><U0020><U0025><U0054><U0025>/
+<U004E><U0025><U0063><U0025><U004E>"
+country_num   616
+country_name  "<U0050><U006F><U006C><U0073><U006B><U0061>"
+country_ab2   "<U0050><U004C>"
+country_ab3   "<U0050><U004F><U004C>"
+country_post  "<U0050><U004C>"
+country_car   "<U0050><U004C>"
+country_isbn  "<U0038><U0033>"
+lang_name     "<U006B><U0061><U0073><U007A><U00EB><U0062><U0073><U0063><U007A><U0069>"
+lang_ab       ""
+lang_term     "<U0063><U0073><U0062>"
+lang_lib      "<U0063><U0073><U0062>"
+END LC_ADDRESS
+
+LC_TELEPHONE
+copy "pl_PL"
+END LC_TELEPHONE
+
+LC_MEASUREMENT
+copy "pl_PL"
+END LC_MEASUREMENT
Index: glibc-2.3.5/localedata/SUPPORTED
===================================================================
--- glibc-2.3.5.orig/localedata/SUPPORTED
+++ glibc-2.3.5/localedata/SUPPORTED
@@ -71,6 +71,7 @@
 ca_IT/ISO-8859-15 \
 cs_CZ.UTF-8/UTF-8 \
 cs_CZ/ISO-8859-2 \
+csb_PL/UTF-8 \
 cy_GB.UTF-8/UTF-8 \
 cy_GB/ISO-8859-14 \
 da_DK.UTF-8/UTF-8 \
